resources/ext/ezbake.conf

```diff
@@ -7,8 +7,7 @@
 ezbake: {
    pe: {}
    foss: {
-      redhat: { dependencies: ["puppet >= 3.7.3"
-                               "puppet < 4.0.0"
+      redhat: { dependencies: ["puppet-agent >= 0.2.0"
```

Comment:
Do we actually need to specify a version here? Or will just `puppet-agent` work?